openpype/hosts/maya/plugins/publish/validate_rig_out_set_node_ids.py

```diff
@@ -51,10 +52,12 @@ def get_invalid(cls, instance):
                          noIntermediate=True)
 
         for shape in shapes:
-            history_id = lib.get_id_from_history(shape)
-            if history_id:
+            sibling_id = \
+                lib.get_id_from_sibling(shape,
+                                        history_only=cls.allow_history_only)
```

Comment:
Should this maybe change in code style to the following?
```
sibling_id = lib.get_id_from_sibling(
    shape,
    history_only=cls.allow_history_only
)
```
I don't recall seeing much of the code style in OP code base that you're using now. (I might be wrong)

Reply:
Actually - what you're showing there would be even better. But I believe it was moved with `\` due to the exceeding the PEP08 line length. In that case the code style I showed in my comment I think is what is preferred here.